src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
-body {
+@import './variables.scss';
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
```

Comment:
create a separate file for bem block

Reply:
checklist point
<img width="645" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/102411130/e65be796-3f6c-4a46-8797-acda2915e8db">